ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/EthProtocolConfiguration.java

```diff
@@ -14,169 +14,16 @@
  */
 package org.hyperledger.besu.ethereum.eth;
 
-import org.hyperledger.besu.util.number.PositiveNumber;
-
-import java.util.Objects;
-
-import com.google.common.base.MoreObjects;
+import lombok.Builder;
+import lombok.Value;
 
+@Value
+@Builder
 public class EthProtocolConfiguration {
-
-  public static final int DEFAULT_MAX_GET_BLOCK_HEADERS = 192;
```

Comment:
I think we should keep the static constant variables.  There are many places where we create a builder just to peek the default value.  This should be a true constant.

Reply:
Done